.travis.yml

```diff
@@ -3,34 +3,41 @@ sudo: false
 language: python
 
 addons:
-    firefox: "52.0esr"
+    firefox: "52.0.1"
 
 python:
     - "2.7"
 
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=quality
+
 branches:
     only:
       - master
 cache:
     directories:
         - $HOME/.cache/pip
 
-before_install:
-    - "export DISPLAY=:99"
+before_script:
+    - "export DISPLAY=:99.0"
     - "sh -e /etc/init.d/xvfb start"
+    - "sleep 3" # give xvfb some time to start
+
+before_install:
     - "wget https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz"
     - "mkdir geckodriver"
     - "tar -xzf geckodriver-v0.15.0-linux64.tar.gz -C geckodriver"
     - "export PATH=$PATH:$PWD/geckodriver"
     - "pip install -U pip"
 
 install:
-    - "make install"
-    - "pip install coveralls"
+    - "pip install tox-travis"
 
 script:
-    - "make quality"
-    - "make cover"
+    - tox
 
 after_success:
+    - "pip install coveralls"
```

Comment:
This probably isn't the right PR to fix it in, but we want to gradually get everything moved from coveralls to codecov.